### PR TITLE
Keep original content in case we want to output the whole content

### PR DESCRIPTION
**Pull request by huali027**
- fix: #2375 
Signed-off-by: Huanhuan Li <huali@redhat.com>

### CODE REVIEW

File: insights/parsers/hammer_ping.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def parse_content(self, content):
         self.status_of_service = {}
         self.response_of_service = {}
         self.errors = []
-
+        self.raw_content = content
```
Comment (xiangce) [inline insights/parsers/hammer_ping.py]: Please add the corresponding docstring for the new attribute.

**Comment by huali027** [comment]
> @huali027 - Which issue does this PR fix?

do we need to create an issue in insights-core for every PR?

File: insights/parsers/hammer_ping.py (hunk 1 above)
Comment (huali027) [reply insights/parsers/hammer_ping.py]: > Please add the corresponding docstring for the new attribute.

sure, I'll add it in next commit

**Comment by xiangce** [comment]
> > @huali027 - Which issue does this PR fix?
> 
> do we need to create an issue in insights-core for every PR?

For bugfix, enhancement, Yes. Not mandatory for new parsers, but it's recommended.

**Comment by huali027** [comment]
> > > @huali027 - Which issue does this PR fix?
> > 
> > 
> > do we need to create an issue in insights-core for every PR?
> 
> For bugfix, enhancement, Yes. Not mandatory for new parsers, but it's recommended.

sure, I'll create one, thanks.

**Comment by huali027** [comment]
> Approved pending an issue for tracking

I have created an issue https://github.com/RedHatInsights/insights-core/issues/2375,
and I added fixes in the second commit

**Comment by xiangce** [comment]
@huali027 - Do not paste the issue number to git commit, since no link-ship will be created in that way. and reviewers cannot get anything.

**Pull request merged**